### PR TITLE
Include a DB Docker container and a Makefile

### PR DESCRIPTION
**Pull request by patrislav**
## Abstract

This PR improves end-to-end testing by including a `docker-compose.yml` file and a `Makefile` to easily spin up a local environment.

## Metadata

Related: #21

### CODE REVIEW

File: scripts/migrate.go [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+	_ "github.com/go-sql-driver/mysql"
+
+	"github.com/cycloidio/terracost/mysql"
+)
+
+func main() {
+	db, err := sql.Open("mysql", "root:terracost@tcp(localhost:33060)/terracost_test?multiStatements=true")
+	if err != nil {
+		panic(err)
```
Comment (xescugc) [inline scripts/migrate.go:16]: Change it for a `log.Fatalf` I would say.

Same for the next one.

**Review by xescugc — CHANGES_REQUESTED** [review]

File: scripts/migrate.go (hunk 1 above)
Comment (patrislav) [reply scripts/migrate.go]: Changed

**Review by xescugc — COMMENTED** [review]
rs

**Comment by patrislav** [comment]
Rebased

**Review by xescugc — APPROVED** [review]

**Pull request merged**